src/routes/forumRoutes.js

```diff
@@ -38,5 +45,10 @@ router
   .put(updateReply)
   .delete(deleteReply);
 
+// vote routes
+router.post("/vote/", addVote);
+router.route("/vote/:content_id").get(getVotes).put(updateVote);
+router.delete("/vote/:content_id/:vote_id", removeVote);
```

Comment:
pass `content_id` and `vote_id` as query parameter
`?content_id=some-id&vote_id=some-vote-id`